scss/lessonCard.scss

```diff
@@ -0,0 +1,21 @@
+.card-title {
+  font-size: 1.5rem;
+  font-weight: bold;
+  line-height: 1.33;
+}
+.icon-messages {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 75%;
+  justify-content: space-between;
+  white-space: nowrap;
+}
+.icon-text {
```

Comment:
according to BEM, this should be `.lesson-card__icon-text`

Reply:
Fixed 👍